react/features/base/dialog/components/native/styles.js

```diff
@@ -117,7 +117,7 @@ const brandedDialogText = {
 };
 
 const brandedDialogLabelStyle = {
-    color: schemeColor('text'),
+    color: ColorPalette.white,
```

Comment:
Please replace the ColorPalette object with the new design Tokens. 
You can search for the Tokens.js file for a full view and see other references.

Reply:
Not sure how to do that. Tokens seem to be part of the theme so they can be used inside of a component, not a style file